src/infection-model/rt.tsx

```diff
@@ -0,0 +1,107 @@
+import { ascending } from "d3-array";
+import { fromUnixTime } from "date-fns";
+import mapValues from "lodash/mapValues";
+
+import { getFacilityModelVersions } from "../database";
+import { totalConfirmedCases } from "../impact-dashboard/EpidemicModelContext";
+import { Facility } from "../page-multi-facility/types";
+
+type RawRtRecord = {
+  date: number; // timestamp
+  value: number;
+};
+
+type RawRtData = {
+  Rt: RawRtRecord[];
+  low90: RawRtRecord[];
+  high90: RawRtRecord[];
+};
+
+type ErrorResponse = {
+  error: string;
+};
+
+type RtInputs = {
+  dates: number[];
+  cases: number[];
+};
+
+export type RtRecord = {
+  date: Date;
+  value: number;
+};
+
+export type RtData = {
+  Rt: RtRecord[];
+  low90: RtRecord[];
+  high90: RtRecord[];
+};
+
+const isError = (obj: RawRtData | ErrorResponse): obj is ErrorResponse => {
+  return (obj as ErrorResponse).error !== undefined;
+};
+
+const getFetchUrl = () => {
+  let url = "https://us-central1-c19-backend.cloudfunctions.net/calculate_rt";
```

Comment:
Let's add a TODO here to replace this with the appropriate env var, unless that's coming in _this_ PR.

Reply:
This one is already working as intended and shouldn't have to change. It's just keyed off the name of the environment. There will be an additional flag that we will use to decide whether we actually _use_ the UI components that will invoke this function (regardless of environment)